server/deployment/src/main/java/io/quarkiverse/openapi/server/generator/deployment/codegen/ApicurioOpenApiServerCodegen.java

```diff
@@ -94,7 +95,12 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
 
     private File convertToJSON(Path yamlPath) throws CodeGenException {
         try {
-            ObjectMapper yamlReader = new ObjectMapper(new YAMLFactory());
+            LoaderOptions loaderOptions = new LoaderOptions();
+            loaderOptions.setCodePointLimit(12 * 1024 * 1024); // 12 MB
```

Comment:
Can this value be configurable with the previous limit? If there are larger spec files, this limit will not be sufficient.

Reply:
The previous limit was 3MB and was set by default.  I used a benchmark 'large' size, I guess it could be doubled or set to the integer maximum. 
Do you think this limit should be decided dynamically?
@masayag

Reply:
I would like to left my opnion about it, I think we can use a configuration for `org.yaml.snakeyaml.LoaderOptions`